assets/js/blocks/mini-cart/frontend.ts

```diff
@@ -152,4 +152,18 @@ window.addEventListener( 'load', () => {
 			);
 		}
 	} );
+
+	const head = document.head || document.getElementsByTagName( 'head' )[ 0 ];
+	const style = document.createElement( 'style' );
+	const body = document.body || document.getElementsByTagName( 'body' )[ 0 ];
+	const backgroundColor = getComputedStyle( body ).backgroundColor;
```

Comment:
What do you think if, instead of only copying `backgroundColor` we copy the entire `background` declaration? In some style variations it makes a difference (ie: Pilgrimage and the ones which have gradients). But I wonder if there are some risks from this approach. :thinking:

Reply:
The risk here is the background can override the background color set by users. For example, gradient uses background-image, which will display regardless of the background color.

Reply:
Oh, right. Good point!